oss-internship-2020/curl/examples/example3.cc

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sandboxed version of simplessl.c
+// HTTPS GET request
+
+#include <cstdlib>
+
+#include "../sandbox.h"
+
+class CurlSapiSandboxEx3 : public CurlSapiSandbox {
+ public:
+  CurlSapiSandboxEx3(std::string ssl_certificate, std::string ssl_key,
+                     std::string ca_certificates)
+      : ssl_certificate(ssl_certificate),
+        ssl_key(ssl_key),
+        ca_certificates(ca_certificates) {}
+
+ private:
+  std::unique_ptr<sandbox2::Policy> ModifyPolicy(
+      sandbox2::PolicyBuilder*) override {
+    // Add the syscalls and files missing in CurlSandbox to a new PolicyBuilder
+    auto policy_builder = std::make_unique<sandbox2::PolicyBuilder>();
+    (*policy_builder)
+        .AllowFutexOp(FUTEX_WAIT_PRIVATE)
+        .AllowGetPIDs()
+        .AllowGetRandom()
+        .AllowHandleSignals()
+        .AllowSyscall(__NR_sysinfo)
+        .AddFile(ssl_certificate)
+        .AddFile(ssl_key)
+        .AddFile(ca_certificates);
+    // Provide the new PolicyBuilder to ModifyPolicy in CurlSandbox
+    return CurlSapiSandbox::ModifyPolicy(policy_builder.get());
+  }
+
+  std::string ssl_certificate;
+  std::string ssl_key;
+  std::string ca_certificates;
+};
+
+int main(int argc, char* argv[]) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+  google::InitGoogleLogging(argv[0]);
+
+  absl::Status status;
+  sapi::StatusOr<int> status_or_int;
+  sapi::StatusOr<CURL*> status_or_curl;
+
+  // Get input parameters (should be absolute paths)
+  if (argc != 5) LOG(FATAL) << "wrong number of arguments (4 expected)";
+  std::string ssl_certificate = argv[1];
+  std::string ssl_key = argv[2];
+  std::string ssl_key_password = argv[3];
+  std::string ca_certificates = argv[4];
+
+  // Initialize sandbox2 and sapi
+  CurlSapiSandboxEx3 sandbox(ssl_certificate, ssl_key, ca_certificates);
+  status = sandbox.Init();
+  if (!status.ok())
```

Comment:
Wrap the `LOG` statement in a `{}`-block. Also below (and in other files).

The style guide also allows statements on the same line as the `if` to omit the braces, but I personally just always add them unconditionally.